protected/humhub/modules/content/components/ContentContainerModuleManager.php

```diff
@@ -21,6 +21,20 @@
  */
 class ContentContainerModuleManager extends \yii\base\Component
 {
+    /**
+     * @var integer Module option "set as default" is in activated state
+     */
+    const STATE_ACTIVATED = 0;
```

Comment:
Maybe we should use those constants in `@admin/views/module/setAsDefault.php` and `@admin/models/forms/ModuleSetAsDefaultForm.php`

Reply:
Remove these constants. It's redundant.